src/Skyroom/Adapter/WooCommerceAdapter.php

```diff
@@ -363,7 +363,7 @@ function processOrder($orderId, $order, UserRepository $userRepository, EventRep
                         continue;
                     }
 
-                    $userRepository->addUserToRoom($user, $product->get_skyroom_id(), $orderId);
+//                    $userRepository->addUserToRoom($user, $product->get_skyroom_id(), $orderId);
```

Comment:
We don't need the whole method anymore. no need to do anything at processing order.

Reply:
Ok